README.md

```diff
@@ -79,7 +99,7 @@ Then manually install the following JARs:
 
 Twitter APIs support three types of authentications:
 
-* OAuth 2.0 Authorization with PKCE
+* #### OAuth 2.0 Authorization with PKCE
```

Comment:
In the first commit, I put a colon after the little title, I removed and it might work now